src-built-in/adapters/indexedDBAdapter.js

```diff
@@ -288,7 +288,7 @@ const IndexedDBAdapter = function () {
 		const request = objectStore.get(combinedKey);
 
 		request.onsuccess = (event) => {
-			const data = event.target.result && event.target.result.value ? event.target.result.value : {};
+			const data = event.target.result && event.target.result.value ? event.target.result.value : undefined;
```

Comment:
@NinjaManatee it feels like this code should be:
```
let data;
if(event.target.result) data = event.target.result.value
```

Currently you're forcing "null" to be "undefined". I think this is an artifact of how the code has evolved from originally forcing it to {}.

Reply:
Yeah, I did the minimum change, but didn't consider converting null to undefined. I think this would do what you want as well:
```
const data = event.target.result ? event.target.result.value : undefined;
```

If value exists, null or otherwise, its value would be used, but if result doesn't exist the default value is undefined. The assumption is that target will always exist, but I'm not 100% sure whether that is a valid assumption.